calicoctl/calico_ctl/status.py

```diff
@@ -252,3 +273,42 @@ def pprint_bird_protocols(version):
         print str(x) + "\n"
     else:
         print results + "\n"
+
+
+def _gobgp(cmd):
+    if getattr(sys, 'frozen', False):
+        gobgp = sys._MEIPASS + "/gobgp"
+    else:
+        gobgp = "gobgp"
+    results = subprocess32.check_output([gobgp, "-j"] + cmd.split(" "))
+    return json.loads(results)
+
+
+def pprint_gobgp_protocols(version):
+    x = PrettyTable(["Peer address", "Peer type", "State", "Since", "Info"])
+
+    now = int(time.mktime(datetime.datetime.now().timetuple()))
+    try:
+        neighbors = _gobgp("neighbor -t ipv%d" % version)
+    except subprocess32.CalledProcessError:
```

Comment:
@ishidawataru: it wasn't running when I checked yesterday which I thought was weird, and didn't see anything listening on port 50051 in the caalico/node container either. Weirder thing is BIRD and BIRD6 deamons were running eventough I selected `--backend=gobgp`

Anyway, I'll try with that bugfix and let you know.